### PR TITLE
fix issue 430 400

### PR DESCRIPTION
**Pull request by tomdong2010**

### CODE REVIEW

File: internal/qkcapi/api.go [hunk 1]
```diff
@@ -784,9 +784,14 @@ func (e *EthBlockChainAPI) GetBalance(address common.Address, fullShardKey *hexu
...
 }
+func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey string) (hexutil.Uint64, error) {
 
-func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey *hexutil.Uint) (hexutil.Uint64, error) {
-	fullShardId, err := getFullShardId(fullShardKey)
+	fullShard, err := strconv.ParseUint(fullShardKey, 0, 16)
+	if err != nil {
+		return 0, err
+	}
+	fullShardUnit := uint(fullShard)
```
Comment (pingke) [inline internal/qkcapi/api.go]: clusterCfg.Quarkchain.GetFullShardIdByFullShardKey(uint32(fullShard))

File: internal/qkcapi/api.go [hunk 2]
```diff
@@ -785,8 +785,13 @@ func (e *EthBlockChainAPI) GetBalance(address common.Address, fullShardKey *hexu
 	return (*hexutil.Big)(balance), nil
 }
 
-func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey *hexutil.Uint) (hexutil.Uint64, error) {
-	fullShardId, err := getFullShardId(fullShardKey)
+func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey string) (hexutil.Uint64, error) {
```
Comment (mask-pp) [inline internal/qkcapi/api.go]: fullShardKey must be is optional, and the version is that hexutil.Uint64 don't support 0x00XX type.

Comment (tomdong2010) [reply internal/qkcapi/api.go]: @mask-pp if change hexutil  ,i  agree

Comment (tomdong2010) [reply internal/qkcapi/api.go]: ok

**Pull request merged**